### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Wallian169**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+                28,
+                [3, 2]
+            ),
+            (
+                100,
+                100,
+                [21, 17]
+            )
+        ]
+    )
```
Comment (Dimosphen1) [inline app/test_main.py:50]: Add tests to test against numbers that are different for `cat_age` and `dog_age` and for negative numbers

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+    def test_occurrences(
+            self,
+            cat_age: int,
+            dog_age: int,
+            result: list
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == result
```
Comment (Dimosphen1) [inline app/test_main.py:57]: Add tests for testing against raised errors in case incorrect input is passed, such as strings, dicts, etc. as mentioned in the checklist. You could create another function for this case

File: app/test_main.py [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
-# write your code here
+
+class TestAllCases:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, result", [
+            (
+                0,
+                0,
+                [0, 0]
+            ),
```
Comment (Dimosphen1) [inline app/test_main.py:13]: Improve the readability by placing arguments on one line
```suggestion
            (0, 0, [0, 0]),
```

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

**Review by pavlopro — APPROVED** [review]
Good Job!
